### DISCUSSION TITLE
Adding tzdata into the local_requirements

### DISCUSSION THREAD

**Discussion by nikos82** [netbox-community/netbox#10015] [Q&A]
Hello,

Could someone explain me how to install the tzdata package into the local_recuirements ? 
Essentially I need the whole process (step by step) in order to achieve this.

I am facing an issue running the upgrade.sh on Ubuntu 20.04 with Python3.8

Thanks in advance for the assistance

**Comment by candlerb** [comment]
What exactly have you tried?  A step-by-step process will sound childish:

1. Choose a text editor that you're comfortable with, e.g. "nano"
2. Start the editor: `nano /opt/netbox/local_requirements.txt` (this file won't exist by default)
3. Enter the line of text: `pytz==2022.1` (*)
4. Save and exit (in nano this is ctrl-X, then Enter to confirm)

(*) Note: there is no need to put tzdata into `local_requirements.txt`, because it already exists in the main `requirements.txt` file.  I am guessing you are talking about pytz instead, as in #9986.

**Comment by nikos82** [comment]
Hi, 

Thanks for your quick response. 
I fixed it checking the #9986 

Apologise for the step-by-step

**Reply by candlerb** [reply]
> Apologise for the step-by-step

OK, I'm sorry :-)

**Comment by mtinberg** [comment]
Y'all jest but we've seen questions on this list from people tasked with managing Netbox whose experience level with Python software would benefit from this kind of detailed instruction, even though this seems obvious to most list members, there are always new people who don't have that experience and Netbox is their first Python app install.

"Press 'X' to Jump" is actually useful to some.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
